### PR TITLE
[terra-framework] clinical-lowlight-theme port

### PR DESCRIPTION
**Pull request by avinashg1994**
## Summary
Ported **Clinical Low Light Theme** to "Terra-Framework".

## Travis Format
**A. Pack default theme assets**
**B. Run wdio in parallel. Split by form factor.**
  1. tiny
  2. small
  3. medium
  4. large
  5. huge
  6. enormous

**C. Pack fusion theme assets**
**D. Run wdio in parallel. Split by form factor.**
  1. tiny
  2. small
  3. medium
  4. large
  5. huge
  6. enormous

## WDIO Format
**The below mentioned commands are applicable at the root level as well as the package level**
1. "**wdio-default**" - Runs the wdio tests only for **Default Theme**.
2. "**wdio-lowlight**" - Runs the wdio tests only for **Clinical Low Light Theme**.
3. "**wdio**" - Runs the wdio tests for both **Default Theme** and **Clinical Low Light Theme**

## Additional/New Files
1. config/wdio/clinical-lowlight-theme/wdio.conf.js - Custom wdio config for clinical-lowlight-theme which disables the color-contrast axe rules.
2. packages/{package-name}/clinical-lowlight-wdio.conf.js - Custom wdio config for clinical-lowlight-theme at the package level to disable color contrast axe rules.

## Additional Details
Accessibility test (color contrast rules) are disabled for clinical-lowlight-theme

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -31,3 +31,14 @@ services:
     depends_on:
       - standalone-chrome
     command: 'npm run wdio-default'
+
+  test-ci-lowlight:
```
Comment (emilyrohrbough) [inline docker-compose.yml]: Lets remove this custom service and update the travis commands to run:


```
script: travis_wait docker-compose run -e FORM_FACTOR=enormous test-ci -sh 'npm run wido-lowlight'

```

Comment (avinashg1994) [reply docker-compose.yml]: Addressed it here : https://github.com/cerner/terra-framework/pull/978/commits/57fd4a3318250ef2f905377a4bcd0f77c32bbab3

**Comment by avinashg1994** [comment]
This PR is up to date with the following changes:

1. All the clinical-lowlight-theme wdio screenshots have been removed.
2. Clinical-lowlight-theme wdio tests are commented out on Travis to improve build times.
Note : This will be enabled once the screenshots have been generated
3. Updated to use existing "test-ci" service.

**Comment by neilpfeiffer** [comment]
Removing label for UX-Review as tech-only PR. (Has been previously validated in pre-port internal theme repo).

UX will continue to monitor as teams start to use and validate for missing theme items or mistakes.

**Pull request merged**